docs/fieldnorm/control_of_documents.md

```diff
@@ -1,25 +1,91 @@
 # Control of documents
 
-Bioinformaticians work in a git environment. Git is a distributed version control system that tracks changes in any set of computer files, usually used for coordination work among programmers collaboratively developing source code during software development. Software procedures and related documentation needs to be controlled, and according to ISO-15189, this can be in any form or type of medium. For software development, there are several reasons to not include all the software documentation within the medical laboratories chosen quality management system.
+## Why do Bioinformaticians need a different QMS
 
-- Software development is done in a distributed version control system. These systems are used for coordination work among programmers. Often these systems also allow for documentation of procedures. With the following advantages a. Automatically link to i.e. code, issues, releases, and release notes. b. Possibility to include code blocks that allow for copy-paste into the console/terminal. It’s fast and readable for bioinformaticians. For example, code blocks can be used for general procedures to start a pipeline, or for trouble shooting.
-- Software development, and hence, corresponding documentation is prone to frequent updates. With the QMS of the laboratory, there are often waiting steps for different members of the laboratory to verify or authorize documentation. Hence, there might be a delay in release of a new version as the documentation of the previous release is not authorized yet.
-- add more
+_Bioinformaticians_ work in a fairly different environement than biologists, and the classic Quality Management System (QMS) offered to biologists are often impractical for _Bioinformaticians_ needs. Here is a list of the most wanted features :
```

Comment:
I would leave out the list. There is no need to specify in what ways we are different from the lab people. We are different, the field norm is how we do things.

Reply:
Mhhh, I am a litte bit puzzled at this, as I thought we had more or less agreed on actually making this list ? Maybe better to discuss it altogether at the next meeting !